jina/flow/__init__.py

```diff
@@ -604,7 +603,7 @@ def index_files(self, patterns: Union[str, List[str]], recursive: bool = True,
                     **kwargs):
         """ Use a set of files as the index source for indexing on the current flow
 
-        :param patterns: The pattern may contain simple shell-style wildcards, e.g. '*.py', '[*.zip, *.gz]'
+        :param patterns: The pattern may contain simple shell-style wildcards, e.g. '\*.py', '[\*.zip, \*.gz]'
```

Comment:
what is `\`??

Make sure to first read sphinx docs before you change any formatting

https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html
https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html

Reply:
This is to escape the `*`. In the online doc, `*` is interpreted literally. 

[https://docs.jina.ai/api/jina.flow.html?highlight=flow#jina.flow.Flow.index_files](https://docs.jina.ai/api/jina.flow.html?highlight=flow#jina.flow.Flow.index_files)